### PR TITLE
feat: add single product page

### PR DESCRIPTION
**Pull request by simenandre**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,2 +1,3 @@
 engine-strict=true
 resolution-mode=highest
+auto-install-peers=true
```
Comment (moripen) [inline .npmrc:3]: What does this do?

File: src/data/projects/get-projects.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+export async function getProjects(): Promise<Project[]> {
+	const dataFromSanity = await client.fetch(query);
+	const projects = z
+		.array(projectModel)
+		.parse(dataFromSanity)
+		.map((project) => {
+			const customer = project.customer?.privacy?.hideName
+				? {
+						...project.customer,
+						name: project.customer?.privacy?.anonymizedName ?? 'Anonymt'
```
Comment (moripen) [inline src/data/projects/get-projects.ts:48]:
```suggestion
						name: project.customer?.privacy?.anonymizedName ?? 'Anonym'
```

**Review by braaar — APPROVED** [review]

**Review by moripen — COMMENTED** [review]
Could you post a screenshot to showcase the changes on the website? It was a bit hard to get an overview of how this would look in my head.

File: src/data/sanity-client-browser.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import imageUrlBuilder from '@sanity/image-url';
+import type { SanityImageSource } from '@sanity/image-url/lib/types/types';
+
+// Get a pre-configured url-builder from your sanity client
+const builder = imageUrlBuilder({
+	projectId: '1zkk2ha0',
+	dataset: 'production'
+});
```
[the remark this answers is not in the document]
Comment (simenandre) [reply src/data/sanity-client-browser.ts:8]: Nope, not a secret :)

**Comment by simenandre** [comment]
> Could you post a screenshot to showcase the changes on the website? It was a bit hard to get an overview of how this would look in my head.

The pages are not linked to, so the visual aspect of it can be seen once it's landed.

File: .npmrc (hunk 1 above)
Comment (simenandre) [reply .npmrc:3]: Just a random fixup. One of those _since I was there's_.

**Pull request merged**